docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -116,3 +116,14 @@ the maximum number of joins that can be reordered at once.
 
 Reduces number of rows produced by joins when optimizer detects that duplicated
 join output rows can be skipped.
+
+``optimizer.table-scan-node-partitioning-min-bucket-to-task-ratio``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``double``
+* **Default value:** ``0.5``
+
+Specifies minimal bucket to task ratio that has to be matched or exceeded in order
+to use table scan node partitioning. When the table bucket count is small
+compared to number of nodes then the table scan will be distributed
```

Comment:
to the number of workers, then the table scan is distributed across all workers for improved parallelism.